common/utils/src/main/resources/error/README.md

```diff
@@ -41,7 +41,7 @@ Unfortunately, we have historically used the term "error class" inconsistently t
 
 Fixing this will require renaming `SparkException.errorClass` to `SparkException.errorCondition` and making similar changes to `ErrorClassesJsonReader` and other parts of the codebase. We will address this in [SPARK-47429]. Until that is complete, we will have to live with the fact that a string like `DATATYPE_MISSING_SIZE` is called an "error condition" in our user-facing documentation but an "error class" in the code.
 
-For more details, please see [SPARK-46810][SPARK-46810].
```

Comment:
After remove last `[SPARK-46810]`, we can still click on it.
<img width="292" alt="image" src="https://github.com/apache/spark/assets/15246973/bd090b47-2ad7-4a3a-ba96-710356f213d2">

Reply:
Oh, interesting.